test/org/javarosa/test/utils/Utils.java

```diff
@@ -0,0 +1,24 @@
+package org.javarosa.test.utils;
+
+import java.io.BufferedInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
+public class Utils {
+
+    private Utils() {
+    }
+
+    public static String convertToString(InputStream inputStream) throws IOException {
+        BufferedInputStream bis = new BufferedInputStream(inputStream);
+        ByteArrayOutputStream buf = new ByteArrayOutputStream();
+        int result = bis.read();
+        while (result != -1) {
+            buf.write((byte) result);
+            result = bis.read();
+        }
+        // StandardCharsets.UTF_8.name() > JDK 7
```

Comment:
I don’t understand this. Is it meant to be a comment, or commented.out (pseudo)code?